src/Illuminate/Auth/Notifications/ResetPassword.php

```diff
@@ -44,9 +52,12 @@ public function via($notifiable)
      */
     public function toMail($notifiable)
     {
+        $email = $notifiable->getEmailForPasswordReset();
+        $link = url("password/reset?email={$email}&expiration={$this->expiration}&token={$this->token}");
```

Comment:
@taylorotwell Are you at all into this idea? Switch all `url()` calls with `route()` calls?